.ai/skills/dokan-frontend-dev/SKILL.md

```
@@ -0,0 +1,167 @@
+        return select('dokan/core').getCurrentUser();
+    }, []);
+};
+```
+
+Available: `useCategories`, `useCurrentUser`, `useCustomerById`, `useCustomerSearch`, `useMutationObserver`, `usePermission`, `useProducts`, `ViewportDimensions`.
+
+## State Management
+
+Uses `@wordpress/data` (Redux-like) stores in `src/stores/`:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)